### PR TITLE
CR for Job #20429 - Add "Create account" link to Qt login dialog in Interface

### PR DESCRIPTION
**Pull request by stojce**
Code Review for Job #20429 - Workitem available at https://www.worklist.net/20429

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: d0e2697cbba4961600dc4e810460e9c8fa02eeb0 available at: https://gist.github.com/f0459ad933287cdc7180<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4542.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4542.dmg

**Comment by problem** [comment]
This looks great. Tested on Mac & Win. @birarda this is ready whenever we pull the trigger for open Alpha. Can you do the CR?

File: interface/ui/loginDialog.ui [hunk 1]
```diff
@@ -456,9 +456,12 @@ border-radius: 4px; padding-top: 1px;</string>
            </property>
            <property name="text">
             <string>&lt;style type=&quot;text/css&quot;&gt;
-	a { text-decoration: none; color: #267077;}
+a { text-decoration: none; color: #267077; margin:0;padding:0;}
+#create {font-weight:bold;}
+p {margin:5px 0;}
 &lt;/style&gt;
-&lt;a href=&quot;https://metaverse.highfidelity.com/password/new&quot;&gt;Recover password?&lt;/a&gt;</string>
+&lt;p&gt;&lt;a id=&quot;create&quot; href=&quot;https://metaverse.highfidelity.com/users/new&quot;&gt;Create account&lt;/a&gt;&lt;/p&gt;
```
Comment (birarda) [inline interface/ui/loginDialog.ui]: can you make this `https://metaverse.highfidelity.com/signup`

Comment (stojce) [inline interface/ui/loginDialog.ui]: Changed

**Comment by hifi-gustavo** [comment]
Build console output for commit: 18b3a52f219a20b233dfc10051e80fa14e81134a available at: https://gist.github.com/f0459ad933287cdc7180<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4542.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4542.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e1fba3c0c00185f251477a5877a83609dd799778 available at: https://gist.github.com/f0459ad933287cdc7180<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4542.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4542.dmg

**Comment by birarda** [comment]
@stojce once https://github.com/highfidelity/hifi/pull/4559 is merged can you use that constant for these two URLs?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8514a39812f05481a83f145a4d797f779fcc12b8 available at: https://gist.github.com/f0459ad933287cdc7180<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4542.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4542.dmg

**Comment by stojce** [comment]
@birarda  switched to URL constants

**Pull request merged**